docker/k8s_tools.py

```diff
@@ -41,16 +41,11 @@ def fetch_pserver_ips():
     pserver_ips = [item[1] for item in pod_list]
     return ",".join(pserver_ips)
 
-def fetch_master_ip():
-    while True:
-        label_selector = "paddle-job-master=%s" % PADDLE_JOB_NAME
-        pod_list = fetch_pods_info(label_selector)
-        master_ip = ""
-        if len(pod_list) >=1:
-            master_ip = pod_list[0][1]
-        if master_ip:
-            return master_ip
-        time.sleep(5)
+def fetch_master_ips():
+    label_selector = "paddle-job-master=%s" % PADDLE_JOB_NAME
+    pod_list = fetch_pods_info(label_selector)
+    master_ips = [item[1] for item in pod_list]
+    return ",".join(master_ips)
```

Comment:
It's just for compatibility with multiple master instance, and it's sure we support one master instance for now, i will fix this.